examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>This is just me, but I think that the placement of commas can be improved upon.</p><p> </p><p>In this tutorial, we...</p><p> </p><p>solved classically, but instead...</p><p> </p><p>programming language, linear algebra, and has...</p><p> </p><p>this tutorial, but if you'd like....</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>